scipy/stats/tests/test_multivariate.py

```diff
@@ -1964,6 +1965,40 @@ def test_to_corr(self):
         assert_allclose(m[0,0], 1)
 
 
+class TestRandomDirection:
+    @pytest.mark.parametrize("dim", [1, 2, 3, 4])
+    @pytest.mark.parametrize("size", [5, (5, 4)])
```

Comment:
```suggestion
    @pytest.mark.parametrize("size", [None, 5, (5, 4)])
```
Probably need to adjust the code to accommodate this. If it's easier, you could hard-code the `expected_shape` as part of the parameterization rather than calculating it on the fly.